test/TestRunner.py

```diff
@@ -20,8 +20,7 @@
 import os
 
 import pytest
-
-from ansiblelint import formatters
+import ansiblelint.formatters as formatters
```

Comment:
Local project imports should be separate:
```suggestion

import ansiblelint.formatters as formatters
```